src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -282,19 +305,19 @@
             "type": "object",
             "properties": {
                 "create": {
-                    "$ref": "#/definitions/handlerDefinition"
+                    "$ref": "#/definitions/cudHandlerDefinition"
```

Comment:
See above, we don't want to start schematising specialised handler operations. That's not a good abstraction.